### DISCUSSION TITLE
Separate the issue tracker?

### DISCUSSION THREAD

**Discussion by MasterKia** [uBlockOrigin/uBOL-home#79] [Ideas]
https://github.com/gorhill/uBlock/issues/3726#issue-330818746:
> Reason: the new issue tracker allows me to grant full permissions to volunteers in processing issues without having to grant full commit rights to code repository.

**Comment by gorhill** [comment]
The MV3-related code really is at gorhill/uBlock, here is just to archive the final package assembled as a result of pulling code from gorhill/uBlock, to allow everybody to easily see changes (including rulesets) between versions -- something not really possible from ublock/gorhill.
